voice/voice-ai/x/Actions/ActionsView.swift

```diff
@@ -143,7 +143,8 @@ struct ActionsView: View {
                 switch newPhase {
                 case .active:
                     print("App became active")
-                    SettingsBundleHelper.checkAndExecuteSettings()
+                   // SettingsBundleHelper.checkAndExecuteSettings()
+                  _ = AppSettings.shared
```

Comment:
what is it doing?

Reply:
Initialize properties with default values and setup the listener for any changes.

Reply:
wouldn't this expression do nothing? Since `AppSettings.shared` is not lazy and is initialized long before this